UI/window-basic-settings.hpp

```diff
@@ -408,6 +407,7 @@ private slots:
 
 	void on_disableOSXVSync_clicked();
 
+	void on_choose_clicked_update();
```

Comment:
Avoid naming functions in Qt [auto-connection](https://doc.qt.io/qt-6/qobject.html#auto-connection) style (`on_sender_signal`) if you're not using Qt auto-connection, this will make Qt complain at runtime.

Reply:
Corrected. I don't think the new name I came up with is amazing but maybe it's good enough for now.